tests/contracts/ecdsa/message_2.json

```diff
@@ -0,0 +1,17 @@
+{
+    "_tag": "verify",
```

Comment:
@vaivaswatha So, the goal of  `message_2.json` and `message_3.json` is to change the signature and the message respectively. Shouldn't we add an extra test that passes `verify()`.  How do you get the value of the signature btw? Does it come from `sign_and_verify()`?

Reply:
Good catch. The signatures were from Schnorr, and hence all three of `message_2`, `message_3` and `message_4` were failing. `message_2` was supposed to be a successful verification. `3` and `4` do what you assumed (change message and signature respectively).

I have updated now.

Note that, here, there is no test that dynamically generates a key pair and tests against that. The key pairs are static. With my upcoming PR (to improve test coverage), I have tests that will generate and test on the fly (but not store the key pairs as they can't be tested by a simple text file diff - due to randomness).